developerportal/templates/molecules/recent-work-item.html

```diff
@@ -0,0 +1,46 @@
+{% comment %}
+Inputs:
+  item: StreamBlock
+  block_type: str
+{% endcomment %}
+
+{% load static %}
+{% load wagtailcore_tags %}
+{% load wagtailimages_tags %}
+{% load app_filters %}
+
+{% image resource.image width-480 as img %}
+
+{% if block_type == "video" %}
+  {% static "img/placeholders/video-16-9.jpg" as fallback_url %}
+{% else %}
+  {% static "img/placeholders/article-16-9.jpg" as fallback_url %}
+{% endif %}
+<section class="mzp-c-card mzp-c-card-small mzp-has-aspect-16-9">
+  {% if external_page %}
+    <a href="{{ resource.url }}" class="card-link" data-type="external_page" target="_blank" rel="nofollow noopener">
+  {% else%}
+    <a href="{% pageurl resource %}" class="card-link" data-type="{{ resource.resource_type }}">
+  {% endif %}
+      <div class="mzp-c-card-media-wrapper">
+        <div class="lazy-image-container">
+          <img class="mzp-c-card-image" src="{% firstof img.url fallback_url %}" alt="">
```

Comment:
Is it possible for us to at least provide a `width` for these? It will help with the janky loading nature. Even if we have a base expected width.

https://www.youtube.com/watch?v=4-d_SoCHeWE&feature=youtu.be

Also, I would suggest we add `loading="lazy"` to images we know are likely to be "below the fold". Unless the framework already has that built-in.

Reply:
We don't have lazy-loading of images (yet) - another good one for me to ticket-up. Cheers!